packages/office-ui-fabric-react/src/components/Spinner/Spinner.test.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+
+import { Spinner } from './index';
+
+describe('Spinner', () => {
```

Comment:
Thanks for writing spinner's very first unit test - it is a little odd it is missing the labelPosition but I guess the example tests cover it already.

Reply:
Yea, I think the example tests are _good enough_ for `labelPosition`. A unit test could perhaps check that the default value is what we expect fairly easily, but its not a blocker IMO.